source/strings.tex

```diff
@@ -321,6 +322,7 @@
     using off_type = streamoff;
     using pos_type = u8streampos;
     using state_type = mbstate_t;
+    using comparison_category = strong_ordering;
```

Comment:
Looks like the member aliases for this `char8_t` specialization are missing the alignment with the `=` like the above and below specializations.

Reply:
If any, this is a pre-existing issue. Please create a new editorial issue for that.

Reply:
Filed as #3108.